bin/dispatch-conf

```diff
@@ -434,6 +436,30 @@ class dispatch:
 
         perform_conf_update_session_hooks("post-session")
 
+    def copy_selinux_label(self, curconf, newconf):
+        """Copy the SELinux security label from the current config file to
+        the new/merged config file."""
+        try:
+            label = os.getxattr(curconf, "security.selinux")
+        except OSError as why:
+            if why.errno == 95:
+                # Filesystem does not support xattrs
+                return
```

Comment:
It would be better to use the symbolic ENOTSUP value from the `errno` module. Add `import errno` at the top of the file and make this change:
```suggestion
        except OSError as e:
            if e.errno == errno.ENOTSUP:
                # Filesystem does not support xattrs
                return
```
Also, would it make sense to skip the warning if getxattr sets errno = ENODATA? The man page describes that as "The named attribute does not exist, or the process has no access to this attribute."

I'm not sure if that's a typical data condition on SELinux or not.

Reply:
>I'm not sure if that's a typical data condition on SELinux or not.

The only situation I think this would realistically happen is if someone is in the middle of switching to an SELinux profile but decides to do `etc-update` or `dispatch-conf` before doing a full relabel of all files on the system, or if for some reason they updated the original file without being on an SELinux kernel. In either case this isn't typical during day-to-day SELinux usage, so I think it's worth emitting the warning if it happens.